src/pages/FilesPage/components/FileItem/FileItem.styles.ts

```diff
@@ -1,13 +1,20 @@
 import styled from 'styled-components';
 import Checkbox from '@mui/material/Checkbox';
 import Card from '@mui/material/Card';
+import InsertDriveFileIcon from '@mui/icons-material/InsertDriveFile';
+import Box from '@mui/material/Box';
 
-export const WCard = styled(Card)`
+export const FileIcon = styled(InsertDriveFileIcon)`
+	font-size: 90px;
+	color: #6e52b3;
+	margin-bottom: 6px;
+`;
+
+export const FileCard = styled(Card)`
+	display: inline-block;
```

Comment:
Why don't you use flex instead? What will happen if we have several rows?

Reply:
There is a grid container on top and I wrap these components in the grid item, but you're right, this style is not needed here at all